Change documentation of port number to match fig.yml.

jalessio opened the pull request:
I got a local runbook dev environment running on OS X using boot2docker by following the instructions provided. I came across one discrepancy in the port number of the web app referenced in `INSTALL_DOCKER.md` and have a few other questions which I hope will lead to cleaning up the docs a bit.

I believe the correct port number for accessing the web app is `8000`, not `8080` as defined in [fig.yml](https://github.com/asm-products/cloudroutes-service/blob/master/INSTALLWithDocker.md#start-the-vm).

Questions:
* Since the project ships with a [Vagrantfile](https://github.com/asm-products/cloudroutes-service/blob/master/Vagrantfile), when would you ever want to run the `vagrant init` command referenced in the [Initialize Vagrant](https://github.com/asm-products/cloudroutes-service/blob/master/INSTALLWithDocker.md#initialize-vagrant) section of `INSTALLWithDocker.md`?
* Additionally, the networking for the Vagrantfile is already specified so I'm also wondering if you could delete the entire [Enable a private network](https://github.com/asm-products/cloudroutes-service/blob/master/INSTALLWithDocker.md#enable-a-private-network) section in `INSTALLWithDocker.md`?

madflojo commented:
Good catch on the port number.

As far as the Vagrant version of the install guide I think you are right that we don't necessarily need those anymore after the Vagrantfile was added. So I am good removing those two sections from the Vagrant install guide.

Do you want to go ahead and remove those and commit it to this branch? If you do then go ahead and create a bounty for these minor documentation tweaks and I will merge the PR and award the bounty.

jalessio commented:
Additional changes:
* Removed the unnecessary Vagrant information in `INSTALLWithDocker.md`
* Added Virtualbox as a prerequisite
* Fixed the port number for the web app in `INSTALLWithDocker.md`

I have no idea what to assign for the bounty. I'll leave that to you or we can just skip it on this one.

madflojo commented:
Ok I created one here. https://assembly.com/runbook/bounties/256
